WalletWasabi.Fluent/Views/MainWindow.axaml.cs

```diff
@@ -17,6 +18,16 @@ public MainWindow()
 #if DEBUG
 		AddHandler(KeyDownEvent, KeyDownHandler, RoutingStrategies.Tunnel);
 #endif
+		Activated += MainWindow_Activated;
+	}
+
+	private async void MainWindow_Activated(object? sender, EventArgs e)
+	{
+		var clipboardValue = await ApplicationHelper.GetTextAsync();
+		if (await CoordinatorConfigStringHelper.ParseAsync(clipboardValue) is { } coordinatorConfigString)
+		{
+			await CoordinatorConfigStringHelper.ProcessAsync(coordinatorConfigString);
+		}
```

Comment:
I know it's a draft yet, but use a behavior instead of coding in the code behind file.